src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2729,8 +2730,9 @@ private bool IsMisplacedModifier(SyntaxListBuilder modifiers, SyntaxList<Attribu
                     return true;
                 }
 
-                var incompleteMember = _syntaxFactory.IncompleteMember(attributes, modifiers.ToList(), type.IsMissing ? null : type);
```

Comment:
>type.IsMissing [](http://example.com/codeflow?start=103&length=14)

Could this missing type be  the source of extra errors? Perhaps we should create a new one without them?